apps/website/src/content/docs/toast.mdx

```diff
@@ -1,18 +1,100 @@
 ---
 title: Toast
-description: A Toast is a non modal, unobtrusive window element used to display small amount of information to a user.
+description: A toast is a non-disruptive message that provides feedback based on a user action.
 thumbnail: Toast
 ---
 
 <p>{frontmatter.description}</p>
 
-<Placeholder componentPageName='toasts--positive' />
+## Usage
+
+Before triggering a toast, the toaster needs to be initialized using the `useToaster` hook. This hook returns an object which provides various methods for triggering the toast and changing the toaster settings.
 
 <LiveExample src='Toast.main.jsx'>
   <AllExamples.ToastMainExample client:load />
 </LiveExample>
 
-A toast notification provides a brief message about app processes at the top of the screen. Because these messages are so prominent, we need to be careful not to overuse or misuse them.
+### Status
+
+Toast statuses can be set using the status methods provided by the toaster object. The status method takes in a toast message and an `options` object for further customization. Four available statuses are:
+
+- `positive()`
+- `negative()`
+- `warning()`
+- `informational()`
+
+<LiveExample src='Toast.status.jsx'>
+  <AllExamples.ToastStatusExample client:load />
+</LiveExample>
+
+For further customization with `options`, you can utilize `link`, which allows taking further actions directly from the toast, such as navigating to another page or triggering a specific function.
+
+<LiveExample src='Toast.link.jsx'>
+  <AllExamples.ToastLinkExample client:load />
+</LiveExample>
+
+### Settings
+
+The object returned by `useToaster` also provides a `setSettings` method, which can be used to customize the `placement` and `order` of toasts.
+
+`placement` supports the following values:
+
+- `"top"` (default)
+- `"top-start"`
+- `"top-end"`
+- `"bottom"`
+- `"bottom-start"`
+- `"bottom-end"`
+
+`"start"` indicates the left side of viewport, while `"end"` represents the right side.
+
+<LiveExample src='Toast.placement.jsx'>
+  <AllExamples.ToastPlacementExample client:load />
+</LiveExample>
+
+`order` supports the following values:
+
+- `"auto"` (default)
+- `"descending"`
+- `"ascending"`
+
+When set to `"descending"`, the most recent toasts are on top. When set to `"ascending"`, the most recent toasts are on bottom. When set to `"auto"`, it will behave like `"descending"` when `placement` is set to a value beginning with `"top"`, otherwise it will behave like `"ascending"`.
+
+<LiveExample src='Toast.order.jsx'>
+  <AllExamples.ToastOrderExample client:load />
+</LiveExample>
+
+### Closing toasts
+
+When setting the `hasCloseButton` option available in the status method to `true`, each toast will be displayed with a close button, which allows you to close the toast manually.
```

Comment:
Prefer "end user", because "you" typically refers to the developer who is using our components.

```suggestion
When setting the `hasCloseButton` option available in the status method to `true`, each toast will be displayed with a close button, which allows the end user to close the toast manually.
```

Reply:
I have all the wording/text changes.